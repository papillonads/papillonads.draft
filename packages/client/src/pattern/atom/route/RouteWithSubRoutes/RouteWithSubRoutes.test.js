import React from 'react'
import defaultRender from './RouteWithSubRoutes.render'

jest.mock('react-router-dom', () => {
  return {
    Route: () => <div />,
  }
})

describe('<RouteWithSubRoutes />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
