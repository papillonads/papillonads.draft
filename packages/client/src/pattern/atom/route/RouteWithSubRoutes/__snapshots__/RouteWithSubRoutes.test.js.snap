// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RouteWithSubRoutes /> Snaphot must match routeWithSubRoutesDefault 1`] = `null`;
